shared/src/main/java/uk/gov/di/authentication/shared/entity/AuthenticationAttempts.java

```diff
@@ -0,0 +1,131 @@
+package uk.gov.di.authentication.shared.entity;
+
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbAttribute;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbBean;
+import software.amazon.awssdk.enhanced.dynamodb.mapper.annotations.DynamoDbPartitionKey;
+
+@DynamoDbBean
+public class AuthenticationAttempts {
+
+    private String attemptIdentifier;
+    private String email;
+    private String journeyType;
+    private String authenticationType;
+    private String code;
+    private Integer count;
+    private long timeToLive;
+    private String created;
+    private String updated;
```

Comment:
Is there a way we can mark some of these fields as required?

I would have expected later tests to fail as `authenticationType` is missing (and very much needed).